src/app/screens/Onboard/NewWallet/index.tsx

```diff
@@ -25,15 +28,20 @@ export default function NewWallet() {
     headers.append("Access-Control-Allow-Origin", "*");
     headers.append("Content-Type", "application/json");
 
-    return fetch(`${url}/create`, {
+    return fetch(walletCreateUrl, {
       method: "POST",
       headers,
-      body: JSON.stringify({ partnerid: "bluewallet", accounttype: "common" }),
+      body: JSON.stringify({ email }),
     })
       .then((res) => {
         res.json().then((data) => {
-          if (data.login && data.password) {
-            setLndHubData(data);
+          if (
+            data.lndhub &&
```

Comment:
[SUGGESTION] could be simplified to: `data.lndhub?.login && data.lndhub?.password && data.lndhub?.url`